README.md

```diff
@@ -1,27 +1,113 @@
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-08-15     | 2018-11-08            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-08     |                       |             | 2.x
 
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-To import it into your code: 
-```js
-import Telemarketing from 'vtex.telemarketing/index'
-```
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
+- [Troubleshooting](#troubleshooting)
+- [Tests](#tests)
 
 ## Usage
-You can use it in your code like a React component with the jsx tag: `<Telemarketing />`. 
-```html
-<Telemarketing />
+
+This app uses our store builder with the blocks architecture. To know more about the Store Builder [click here](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object).
+
+We add the telemarketing as a block in our [Store Header](https://github.com/vtex-apps/store-header/blob/master/store/interfaces.json).
+
+To configure or customize this app, you need to import it in your dependencies in `manifest.json`.
+
+```json
+  dependencies: {
+    "vtex.telemarketing": "2.x"
+  }
 ```
 
-DISCLAIMER: This component will only be displayed if the user has the role of `2 - Televendas (Call center operator)` in it´s profile access.
+Then, add `telemarketing` block into your app theme as we do in our [Store theme app](https://github.com/vtex-apps/store-theme/blob/master/store/blocks.json). 
 
-To give an user the permission of call center operator you need to follow this steps on VTEX admin management page:
+:loudspeaker: **Disclaimer:** This component will only be displayed for the users that have the role of `2 - Televendas (Call center operator)` in their access profile.
+
+To give an user the permission of call center operator you need to follow these steps on VTEX admin management page:
 
 - Access the page: Account Management -> Access Profiles -> Click in New Profile -> Select `2 - Televendas (Call center operator)`
 - Add the email of the users that are responsable for impersonating customers(call center operators).
+
+### Blocks API
+
+When implementing this app as a block, various inner blocks may be available. The following interface lists the available blocks within telemarketing and describes if they are required or optional.
+
+```json
+{
+  "telemarketing": {
+    "component": "index"
+  }
+}
+```
+
+As you can see, this app has no required or optional block.
+
+#### Configuration 
+Configuration NDA.
```

Comment:
Is there any configuration in this app? Isn't better to say that it does not have configurations props for now?